packages/celotool/ci_test_governance.sh

```diff
@@ -13,7 +13,7 @@ if [ "${1}" == "checkout" ]; then
     # Test master by default.
     BRANCH_TO_TEST=${2:-"master"}
     echo "Checking out geth at branch ${BRANCH_TO_TEST}..."
-    ../../node_modules/.bin/mocha -r ts-node/register src/e2e-tests/governance_tests.ts --branch ${BRANCH_TO_TEST}
+    ../../node_modules/.bin/mocha --retries 3 -r ts-node/register src/e2e-tests/governance_tests.ts --branch ${BRANCH_TO_TEST}
```

Comment:
I would reduce it to 2, or try how hard is to bail 2 and make a dry run if both failed

Reply:
Thanks Gaston, still looking into whether this will work. Mocha retry will not re-run the `before` step, implying we'll be starting tests with the current state which I don't believe can be expected to pass.